adapter/memquota2/config/config.proto

```diff
@@ -0,0 +1,43 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package adapter.memquota.config;
+
+import "google/protobuf/duration.proto";
+import "gogoproto/gogo.proto";
+
+option go_package="config";
+option (gogoproto.goproto_getters_all) = false;
+option (gogoproto.equal_all) = false;
+option (gogoproto.gostring_all) = false;
+
+message Params {
+	message Quota {
```

Comment:
I've been calling these sub-messages "FooInfo". I think it'd be better if these were named consistently across all of our adapters for this naming. Do we want to call these "Foo" or "FooInfo"? My concern is confusion between instances and these objects if we call them "Foo" (e.g. "Quota" is the name we use for the Istio-thing, and also the name you use in this adapter, but they're not the same thing).

Reply:
Renamed to QuotaInfo